sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/MetricDataPoint.cs

```diff
@@ -10,8 +10,15 @@ internal partial class MetricDataPoint
     {
         public MetricDataPoint(Metric metric, MetricPoint metricPoint)
         {
-            Name = metric.Name;
-            Namespace = metric.MeterName;
+            if (StandardMetricsExtractionProcessor.s_standardMetricNameMapping.TryGetValue(metric.Name, out var metricName))
```

Comment:
What is the reason for this conditional check?

Reply:
OTel does not allow `/` in instrument names. So for `requests/duration` metric I have used a placeholder while creating instrument. That name is replaced back here.

Reply:
We could remove few of the constant tags getting added to histogram and directly add it as a property. As there is a check here it makes it easier to understand for which items we need to add those constant properties.